File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:

        leng=0
        for i in range(len(s)-1,-1,-1):
            if s[i]==" ":
                if leng>0:
                    return leng
                continue
            elif s[i]!=" ":
                leng+=1   

        return leng
        

File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int l=0;
        for(int i=s.size()-1; i>=0;i--){
            if(s[i]!=' '){
                l++; 
            }
            else if( l>0 && s[i]==' '){
                break;
            }
            
        }
        
        return l;
    }
};